xbmc/pvr/addons/PVRClients.h

```diff
@@ -657,6 +659,8 @@ namespace PVR
 
     int GetClientId(const std::string& strId) const;
 
+    bool IsRecordable(const EPG::CEpgInfoTag *tag) const;
```

Comment:
Avoid raw pointers, use shared pointer instead => <code>const EPG::CEpgInfoTagPtr &tag</code>, yes this can be forward declared as well. still no need for the include above.